wca-guidelines.md

```diff
@@ -164,7 +164,8 @@ To be more informative, each Guideline is classified using one of the following
 - A1a4+) [REMINDER] If a competitor has accidentally been permitted to exceed the time limit, the time limit must be enforced retroactively, and the judge, competitor, and WCA Delegate should be informed (see [Regulation 1g2](regulations:regulation:1g2)). Judges must always be aware of the time limit for a current attempt (which might depend on previous attempts, in the case of a cumulative time limit).
 - A2c1+) [REMINDER] In the past, score sheets have been used to cover puzzles while leaving some sides exposed. This is no longer permitted.
 - A2d1+) [ADDITION] If a missing scramble signature is discovered before an attempt, the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
-- A2d1++) [ADDITION] If a missing scramble signature is discovered after the beginning of the attempt, the result may be kept, at the discretion of the WCA Delegate. However, the WCA Delegate should not allow such incidents to become common.
+- A2d1++) [ADDITION] An attempt must not be disqualified solely due to a missing scramble signature. If a scramble signature is missing after the beginning of an attempt, the judge should allow the attempt to continue (if it is still in progress) and the WCA Delegate should allow the result to be kept. However, the WCA Delegate may award an extra attempt if they have a reason (other than the missing signature) to suspect that the scramble was incorrect.
```

Comment:
I agree that a `must` is not proper for a guideline. I think a `should` would also work pretty well here.
```suggestion
- A2d1++) [ADDITION] An attempt should not be disqualified solely due to a missing scramble signature. If a scramble signature is missing after the beginning of an attempt, the judge should allow the attempt to continue (if it is still in progress) and the WCA Delegate should allow the result to be kept. However, the WCA Delegate may award an extra attempt if they have a reason (other than the missing signature) to suspect that the scramble was incorrect.
```

Reply:
Actually thinking a little bit more, I'm happy leaving the must here

Reply:
The regulations currently use the verbs "replace", "grant" and "receive" when referring to "extra". Can "award" be changed to "grant" here? Receiving an extra isn't really an award :-)
Also the regulations never use "however", it seems a bit redundant here.

Reply:
```suggestion
- A2d1++) [ADDITION] An attempt must not be disqualified solely due to a missing scramble signature. If a scramble signature is missing after the beginning of an attempt, the judge should allow the attempt to continue (if it is still in progress) and the WCA Delegate should allow the result to be kept. The WCA Delegate may grant an extra attempt if they have a reason (other than the missing signature) to suspect that the scramble was incorrect.
```

Reply:
A few things:
- Given Delegate feedback, I think it's important to keep the "must".
- We do use "However" (once) in the Guidelines already. I think it's valuable to keep it here to emphasize that this is the exceptional case.
- You're right about "award" — I totally forgot that we stick with "grant" now. However, I think that if there is a good reason to use an extra, we should emphasize that it's not optional, so I've changed it to "replace the attempt with an extra attempt" below. (https://www.worldcubeassociation.org/regulations/#11e3 already covers the case where the extra should be provisional.)
- Based on feedback, changing the judge's action to "must".

```suggestion
- A2d1++) [ADDITION] An attempt must not be disqualified solely due to a missing scramble signature. If a scramble signature is missing after the beginning of an attempt, the judge must allow the attempt to continue (if it is still in progress) and the WCA Delegate should allow the result to be kept. However, the WCA Delegate may replace the attempt with an extra attempt if they have a reason (other than the missing signature) to suspect that the scramble was incorrect.
```